odoo/addons/base/models/ir_qweb.py

```diff
@@ -2508,6 +2509,28 @@ def _get_asset_link_urls(self, bundle, debug=False):
         asset_nodes = self._get_asset_nodes(bundle, js=False, debug=debug)
         return [node[1]['href'] for node in asset_nodes if node[0] == 'link']
 
+    def _pregenerate_assets(self):
+        _logger.info('Pregenerating assets bundles')
+        views = self.env['ir.ui.view'].search([('type', '=', 'qweb'), ('arch_db', 'like', '%t-call-assets%')])
```

Comment:
Same than my previous comment. But in this case, it isn't incorrect but only redundant.